### DISCUSSION TITLE
Email out gives error: email body empty... but it isn't.

### DISCUSSION THREAD

**Discussion by Nasnl** [Budibase/budibase#5884] [Help] [answered]
Hi,

Anyone knows what might be wrong?

I added an email out step and filled the fields:
![image](https://user-images.githubusercontent.com/8427570/168387762-7f6808cf-3af6-4f30-962d-50d665ebb740.png)

When testing, I get a false result on the email step with the error message that the body is not allowed to be empty:

![image](https://user-images.githubusercontent.com/8427570/168387640-11f5d880-f9b6-4b43-bab2-330218b63da8.png)

I assume that the field 'HTML contents' is filling the email body, right?
If so, how can the error be like that?

BTW: this also happens in the same way when I do not use a variable in the 'HTML contents', but just plain text like in the order confirmation example in the docs.

Thanks a lot!

**Comment by Nasnl** [comment]
Maybe in the mean time (while waiting for a solution): I think it is obvious... (but assumptions are the mother of all F* ups), but can someone confirm that the field 'HTML Content' actually fills the email body? ✌

**Comment by melohagan** [comment, marked as answer]
Hey @Nasnl 

I think there's an issue with the `{{ Sender }}` binding. That doesn't look like a valid binding. 

Do you mean something like: `{{ trigger.row.Sender }}` ?

**Reply by Nasnl** [reply]
Thanks @melohagan 

That indeed did the trick. Due to the error message mentioning the empty body as cause, I focused on that one... but indeed it was the way the variable was used.

Appreciate your help!

**Reply by melohagan** [reply]
You're welcome! I agree that the error message is confusing, so I've raised a minor bug here: https://github.com/Budibase/budibase/issues/5905
